src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -768,6 +775,16 @@ export default class OncoprintControls extends React.Component<IOncoprintControl
                         onClick={this.onInputClick}
                     /> Show legends for clinical tracks
                 </label></div>
+                <div className="checkbox" style={{marginLeft:20}}><label>
+                    <input
+                        data-test="onlyShowClinicalLegendsForAltered"
+                        type="checkbox"
+                        value={EVENT_KEY.onlyShowClinicalLegendForAlteredCases}
+                        checked={this.props.state.onlyShowClinicalLegendForAlteredCases}
+                        onClick={this.onInputClick}
```

Comment:
Would onChange be slightly more semantic?

Reply:
I've always been happy with onClick, I think I've had issues with onChange with controlled components